persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -2039,3 +2064,151 @@ postgresMkColumns :: [EntityDef] -> EntityDef -> ([Column], [UniqueDef], [Foreig
 postgresMkColumns allDefs t =
     mkColumns allDefs t
     $ setBackendSpecificForeignKeyName refName emptyBackendSpecificOverrides
+
+-- | Wrapper for persistent SqlBackends that carry the corresponding
+-- `Postgresql.Connection`.
+--
+-- @since 2.13.1.0
+data RawPostgresql backend = RawPostgresql
+    { _persistentBackend :: backend -- ^ The persistent backend
+    , _rawPostgresqlConnection :: PG.Connection -- ^ The underlying `PG.Connection`
```

Comment:
```suggestion
    { persistentBackend :: backend 
    -- ^ The persistent backend
    --
    -- @since 2.13.1.0
    , rawPostgresqlConnection :: PG.Connection 
    -- ^ The underlying `PG.Connection`
    -- 
    -- @since 2.13.1.0
```